src/rtcModule/webrtc.h

```diff
@@ -96,6 +96,7 @@ enum TermCode: uint8_t
     kCallGone = 8,
     kBusy = 9,                  // < Peer is in another call
     kNotFinished = 10,          // < It is no finished value, it is TermCode value while call is in progress
+    kDestroyByCallCollision = 11,// < The call has finished by a call collision
```

Comment:
In case of really need a new termcode, I'd like to avoid any future collision with upcoming termcodes (in case the protocol eventually needs them).

Reply:
Yes, it's right. I modify the termCode value